modules/n1ql/pages/n1ql-language-reference/hints.adoc

```diff
@@ -227,6 +224,23 @@ WHERE sourceairport = "SFO";
 ----
 ====
 
+.Use any suitable Full Text Search index
+====
+Specify that the query service should prefer an FTS index, without specifying the index by name.
+To qualify for this query, there must be an FTS index on state, city, and type, using the keyword analyzer.
```

Comment:
The FTS index doesn't need to index "city" for this case.
Optionally if it was a type-mapped index on "hotel", it doesn't need to have "type" indexed as well.

Reply:
Done